.circleci/config.yml

```
@@ -4,20 +4,103 @@ orbs:
 jobs:
   run_pre_commit:
     docker:
-      - image: quay.io/astronomer/ci-pre-commit:2020-12
+      - image: quay.io/astronomer/ci-pre-commit:2021-04-27
```

We should keep this tag at a less precise tag unless there's a good version to include the day of the month.
```suggestion
      - image: quay.io/astronomer/ci-pre-commit:2021-04
```